mix.exs

```diff
@@ -131,11 +133,20 @@ defmodule Livebook.MixProject do
     version: @version,
     logo_path: "static/images/logo.png",
     url_schemes: ["livebook"],
+    additional_paths: ["/rel/vendor/bin", "/rel/vendor/elixir/bin"],
```

Comment:
I think this is a bit confusing that we have `vendor/elixir/bin` for Elixir and `vendor/bin` for Erlang. In that sense I think what we previously had, `vendor/erts/bin`, made more sense to me.

I think this:

> Or perhaps we should nest all of Erlang under rel/vendor/erlang...

(https://github.com/livebook-dev/livebook/pull/929#discussion_r791008790)

would make even more sense actually. We're vendoring in all of OTP, not just binaries but also libraries. If we do that then `rel/lib` would only contain Livebook and its Mix dependencies but no OTP or Elixir libraries, those would be in vendor. WDYT?

Reply:
> vendor/erts/bin, made more sense to me.

This may not work as the scripts in `erts/bin` may not be the same as in `bin`.

> If we do that then rel/lib would only contain Livebook and its Mix dependencies but no OTP or Elixir libraries

Unfortunately this may not work either as Elixir releases modify the Elixir executables and it can change in further ways in the future.

The current approach does duplicate Elixir but it is fine because it is small. Plus the non-vendored version has stripped beams.